hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -1455,6 +1467,10 @@ private void updateRegionsInTransitionMetrics(final RegionInTransitionStat ritSt
     metrics.updateRITOldestAge(ritStat.getOldestRITTime());
     metrics.updateRITCount(ritStat.getTotalRITs());
     metrics.updateRITCountOverThreshold(ritStat.getTotalRITsOverThreshold());
+    if ((EnvironmentEdgeManager.currentTime() - ritStat.ritThreshold / 2) >= ritStat.statTimestamp){
+      LOG.debug("Oldest RIT hashes and states: "+ritStat.getOldestRITHashesAndStates().toString());
```

Comment:
Since we are anyways exposing rits in metrics, writing DEBUG logs might be overkill I believe. Let's keep the log at TRACE level?